### PR TITLE
Adding 3 Websites

### PR DESCRIPTION
**Pull request by C3n7ral051nt4g3ncy**
### Additions: `3 Sites`

1. **`Naija Planet`**:

- Nigerian Dating 
- User profile example: `https://naijaplanet.com/wales73`

2. **`My Instants`**: 

- Music
- User profile example : `https://www.myinstants.com/en/profile/dave/`

3. `**FCV**`:

- Ukrainian Football (soccer) site
- User profile example: `https://fcv.if.ua/index.php/component/comprofiler/userprofile/Ruslan`

### CODE REVIEW

**Comment by WebBreacher** [comment]
The JSON Schema check failed for the `wmn-data.json` file:
```
 3886 |         },
> 3887 |         {
       |         ^ ☹️  cat is missing here!
  3888 |             "name": "my_instants",
```

File: wmn-data.json [hunk 1]
```diff
@@ -3078,6 +3078,18 @@
...
+       {
+        "name" : "my_instants",
+        "uri_check" : "https://www.myinstants.com/en/profile/{account}/",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : " | Myinstants</title>",
+        "m_string" : "Page not found",
+        "m_code" : 404,
+        "known" : ["daniel01", "dave"],
+        "cate" : "music",
```
Comment (AccentuSoft) [inline wmn-data.json:3090]: This need to be 'cat' instead of 'cate' 🐱

Comment (C3n7ral051nt4g3ncy) [reply wmn-data.json:3090]: @WebBreacher Thanks for the heads up! Will correct now.

**Review by AccentuSoft — APPROVED** [review]
LGTM

**Comment by C3n7ral051nt4g3ncy** [comment]
I have a few more sites to add today, but will wait for `PR decongestion`🚦 :las

**Comment by WebBreacher** [comment]
@C3n7ral051nt4g3ncy it looks like you accidentally submitted the `*min*.json` files with your PR. Please removed those.

**Comment by C3n7ral051nt4g3ncy** [comment]
> @C3n7ral051nt4g3ncy it looks like you accidentally submitted the `*min*.json` files with your PR. Please removed those as they are causing conflicts AND they get auto-generated.

The submissions of the two `min.json` files is done automatically by GitHub Desktop, regardless of the fact that I never touch this files. When I make any kind of additions, I make the change on the `web_accounts_list.json` and then on the `wmn-data.json`

**Comment by C3n7ral051nt4g3ncy** [comment]
I have now removed the files @WebBreacher , let me know if you need anything else done.

**Comment by C3n7ral051nt4g3ncy** [comment]
Closing this PR.
Will Resubmit.

**Pull request closed without merging**